client/ice-servers.go

```diff
@@ -19,7 +20,56 @@ import (
 )
 
 func formGenericIceServers(config *config.ICEConfig) (adapters.IceServersConfig, error) {
-	return adapters.IceServersConfig{}, nil
+	iceServers := []webrtc.ICEServer{}
+	if config.StunEnabled {
+		urls := []string{}
+		for proto, ports := range config.StunPorts {
+			for _, port := range ports {
+				stunProto := "stun"
+				if proto == "tls" {
+					stunProto = "stuns"
+				}
+				urls = append(urls,
+					fmt.Sprintf("%s:%s:%d?transport=%s",
+						stunProto, config.StunHost, port, proto))
+			}
+		}
+		iceServers = append(iceServers,
+			webrtc.ICEServer{
+				URLs:           urls,
```

Comment:
The way we test things is to have one url per `webrtc.ICEServer` which is completely foreign to how you would normally do this - but it means we can test one url at a time. 

I'd likely just move this `iceServers =  append(` to where you create the urls string slice.

Reply:
Fixed in [92c3769](https://github.com/nimbleape/iceperf-agent/pull/13/commits/92c3769726b02e4150035e2220b590f70a48fa32)